### PR TITLE
Project setup (book-detail-2)

### PR DESCRIPTION
**Pull request by dsandoval11**
## Summary
- Completed the project setup
- Added equalizer file
- Book cover image is added

## Screenshot
image

## Trello Card
https://trello.com/c/bHFGoRJ9/27-detalle-de-libro-ii-imagen

### CODE REVIEW

**Comment by dsandoval11** [comment]
Roboto font is added in [PR#4](https://github.com/wolox-training/ds-angularjs/pull/4)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="book">
+      <div>
+        <img
+          class="book-cover"
+          src="./images/book-cover.png"
+          alt="book-cover"
+        />
```
Comment (mcallegari10) [inline index.html:18]:
```suggestion
        <img class="book-cover" src="./images/book-cover.png" alt="book-cover" />
```

No need to write it in multiple lines 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="book">
+      <div>
```
Comment (mcallegari10) [inline index.html:13]: I think this div is not necessary

File: scss/detail.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  .title {
+    font-size: 32px;
+    font-weight: bold;
+    margin-bottom: 52px;
+  }
+
+  .subtitle {
+    font-size: 24px;
+    font-weight: bold;
+    color: #828282;
```
Comment (mcallegari10) [inline scss/detail.scss:27]: Use SASS variables for colors

File: scss/detail.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  .info {
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 36px;
+  }
+
+  .data {
+    font-size: 20px;
+    color: #828282;
```
Comment (mcallegari10) [inline scss/detail.scss:38]: Use SASS variable

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (dsandoval11) [reply index.html:18]: Fixed in #3

File: index.html (hunk 2 above)
Comment (dsandoval11) [reply index.html:13]: Fixed in #3

File: scss/detail.scss (hunk 3 above)
Comment (dsandoval11) [reply scss/detail.scss:27]: Fixed in #2

File: scss/detail.scss (hunk 4 above)
Comment (dsandoval11) [reply scss/detail.scss:38]: Fixed in #2

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**